magento/magento2
# Fixed issue when the preview images navigation is triggered by moving the input filed cursor using arrow keys

### Description

-  Added a more specific selector for keyboard navigation listener

### Fixed Issues
1. magento/adobe-stock-integration#847: The Preview images are changed if move the cursor with arrow keys in File Name field

### Manual testing scenarios 
1. Open Adobe Stock Panel
2. Verify ability to navigate opened preview images using keyboard
3. Click Save Preview for an image for filename prompt to open
4. Verify ability to navigate between characters inside input field when prompt is open

### Contribution checklist
